docs/vz330_mellow/gantry/disassemble.md

```diff
@@ -9,7 +9,13 @@ has_children: false
 permalink: /vz330_mellow/gantry/change_parts
 ---
 
-# STLs
+# Disassemble the PLA gantry parts
+
+The Mellow VzBoT kit comes with the gantry pre-assembled containing printed parts made out of PLA 
+plastic. While the PLA plastic parts are a good starting point and provide you with a clear
+reference as to how the gantry is assembled, it is highly recommended to replace the supplied PLA
+parts with a different material that can withstand higher temperatures. Te PLA parts may warp 
```

Comment:
typo here :)
te -> the

Reply:
Oh -- Having the artificial line length limits isn't really required for our use cases. Any particular reason for using them? 't makes the diff look funny :^)